spec/models/data_request_spec.rb

```diff
@@ -146,7 +147,7 @@
   describe '#request_type' do
     context 'valid values' do
       it 'does not error' do
-        expect(build(:data_request, request_type: 'offender')).to be_valid
+        expect(build(:data_request, request_type: 'all_prison_records')).to be_valid
```

Comment:
There are quite a lot of `calls to `build(` in this file that I think we could refactor to `build_stubbed` which is a quicker in-memory alternative which doesn't call `create` and therefore avoids DB calls ([more info here](https://thoughtbot.com/blog/use-factory-girls-build-stubbed-for-a-faster-test#:~:text=build_stubbed%20is%20the%20younger%2C%20more,that's%20where%20the%20similarities%20end.)). 

I changed all of the builds to build stubbed in this file and the tests passed in 4.84 seconds, with build it took 6.66 seconds.

Reply:
Interesting 🤔 I tried it and not all the objects get built correctly, and there wasn't any significant improvement in the run speed. But I'm really keen to explore this more 👍